server/game/cards/06-WoE/ImperatorDrusilla.js

```diff
@@ -0,0 +1,32 @@
+const Card = require('../../Card.js');
+
+class ImperatorDrusilla extends Card {
+    // Play: Destroy another friendly creature.
+    //
+    // While there are more enemy creatures than friendly creatures,
+    // Imperator Drusilla gains splash-attack 4.
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
+                cardType: 'creature',
+                controller: 'self',
+                cardCondition: (card) => card !== this,
```

Comment:
I think you need to use (card, context) => card !== this

We avoid `this` because of cards such as Mimic Gel

Reply:
Done, thanks!